solidity/random-beacon/contracts/RandomBeaconGovernance.sol

```diff
@@ -808,6 +814,38 @@ contract RandomBeaconGovernance is Ownable {
         newMaliciousDkgResultSlashingAmount = 0;
     }
 
+    /// @notice Begins the minimum stake amount update process.
+    /// @dev Can be called only by the contract owner.
+    /// @param _newMinimumStake New minimum stake amount.
+    function beginMinimumStakeUpdate(uint96 _newMinimumStake)
+        external
+        onlyOwner
+    {
+        /* solhint-disable not-rely-on-time */
+        newMinimumStake = _newMinimumStake;
+        minimumStakeChangeInitiated = block.timestamp;
+        emit MinimumStakeUpdateStarted(_newMinimumStake, block.timestamp);
+        /* solhint-enable not-rely-on-time */
+    }
+
+    /// @notice Finalizes the minimum stake amount update process.
+    /// @dev Can be called only by the contract owner, after the governance
+    ///      delay elapses.
+    function finalizeMinimumStakeUpdate()
+        external
+        onlyOwner
+        onlyAfterGovernanceDelay(
+            minimumStakeChangeInitiated,
+            STANDARD_PARAMETER_GOVERNANCE_DELAY
```

Comment:
I think we should consider it a critical parameter. A lower minimum stake means one can get more seats in the group and slashing amount for not submitting entry on time will be much higher.

Reply:
Let's also remember to list this parameter on the list above `CRITICAL_PARAMETER_GOVERNANCE_DELAY`.

Reply:
3c6c981

> Let's also remember to list this parameter on the list above CRITICAL_PARAMETER_GOVERNANCE_DELAY

Sorry, not sure what you mean by that

Reply:
This list: https://github.com/keep-network/keep-core/blob/3c6c9817deae42caf5edd94a29565b3bd7673015/solidity/random-beacon/contracts/RandomBeaconGovernance.sol#L77-L81

Reply:
ok, done in a4541b7